pycloudflare/models.py

```diff
@@ -113,7 +113,8 @@ def records(self):
         return by_name
 
     def create_record(self, name, type, content, ttl=1, proxied=False,
-                      priority=10):
+                      priority=10, service=None, protocol=None, weight=0,
+                      port=None, target=None):
```

Comment:
@zoidbergwill If we're going to use `**kwargs` should I remove all the optional parameters?